src/libs/credential.ts

```diff
@@ -38,16 +38,20 @@ export interface StoredCredential {
   platform?: string
   last_used?: number // last used epoc time,
   clientExtensionResults?: any
+  dpks?: string[] // Device Public Key,
 }
 
 export async function getCredentials(
   user_id: user_id
 ): Promise<StoredCredential[]> {
-  const results: StoredCredential[] = [];
+  let results: StoredCredential[] = [];
```

Comment:
No need to use `let`

```suggestion
  const results: StoredCredential[] = [];
```